src/neo/SmartContract/Native/CryptoLib.cs

```diff
@@ -50,6 +50,19 @@ public static byte[] Sha256(byte[] data)
             return data.Sha256();
         }
 
+        /// <summary>
+        /// Computes the hash value for the specified byte array using the murmur32 algorithm.
+        /// </summary>
+        /// <param name="data">The input to compute the hash code for.</param>
+        /// <param name="seed">The seed of the murmur32 hash function</param>
+        /// <returns>The computed hash code.</returns>
+        [ContractMethod(CpuFee = 1 << 4)]
```

Comment:
@neo-project/ngd-shanghai could you calculate the fee?

Reply:
Any update?

Reply:
Due to test result, Mur is 4 times faster than SHA， so the price should be 1<<13.